client/src/main/python/slipstream/NodeDecorator.py

```diff
@@ -75,6 +75,7 @@ class NodeDecorator(object):
     CLOUDSERVICE_KEY = 'cloudservice'
     SECURITY_GROUPS_KEY = 'security.groups'
     MAX_PROVISIONING_FAILURES_KEY = 'max-provisioning-failures'
+    NATIVE_CONTEXTUALIZATION_KEY = 'native-contextualization'
```

Comment:
Just a noop comment for the future.  `NodeDecorator` is getting overloaded with too much of semantics.  Consider refactoring by extraction.